app/pb_kits/playbook/pb_card/_card.jsx

```diff
@@ -36,13 +80,30 @@ const Card = ({
     [`highlight_${highlight.color}`]: highlight.color,
   })
 
+  // coerce to array
+  const leChildren = typeof(children) === 'object' && children.length ? children : [children]
```

Comment:
We need to ensure that (if we are going to loop through a *list* of children) we indeed always have a list of children 😀